custom_auth/tests.py

```diff
@@ -33,67 +77,240 @@ def delete(self, *args, **kwargs):
 class AuthenticationTest(JWTTestCase):
```

Comment:
Should there be a naming consistency by appending 'Case' to the class names?

Reply:
It's not required by django or python, but I tend to use the TestCase suffix, because they inherit from TestCase. I seem to have forgotten it a few times. Do you think we should use this suffix (or another one) consistently?

Reply:
A suggestion could be to use TestCase if it is not connected to a view, and ViewTest if it is. (So that UserTest is changed into UserViewTest, and UserTestCase can remain as it is.)

Reply:
I feel that is somewhat arbitrary. I agree with having View in the name of view tests, but the suffix should be either Test or TestCase for all tests

Reply:
Agreed. I vote for TestCase since it seems most conventional.